src/pages/about/our-mission.tsx

```diff
@@ -31,39 +31,38 @@ const OurMission: PageWithLayout = () => {
         </div>
         <div>
           <SubSection header="1. We need more data in our movement.">
-            We can’t realistically track or be confident that the projects and
-            organizations out there right now are as effective as they can be.
-            We strongly believe this is the largest issue in growing our
-            movement, and so we aim to build, utilize, and gather data through
-            our projects and network – and then share that data.
+            To determine effectiveness in projects, campaigns, and our work as a
+            whole, meaningful data needs to be tracked and collected. We
+            strongly believe that a data-driven movement will accelerate and
+            elevate our work, which is why we believe in gathering and analyzing
+            data, as well as making it accessible to others.
           </SubSection>
           <SubSection header="2. We need more competition, too.">
             We strongly believe competition is not only healthy but vital in
-            growing and improving our movements’ effectiveness. Competition puts
-            healthy pressure on other organizations and projects to keep
-            building, improving, and innovating – and gives them more data on
-            what works, and what doesn’t work.
+            improving our movement's effectiveness. Competition applies healthy
```

Comment:
```suggestion
            improving our movement&apos;s effectiveness. Competition applies healthy
```

Reply:
Can't we use ’ like on line 45? @Headscracher

Reply:
That's also possible. It's currently mixed in the repository, so feel free to use both solutions